rfc/graphql-schema/additionalTypeDefs/byPage/quickOrder.ts

```diff
@@ -0,0 +1,159 @@
+export default /* GraphQL */ `
+
+type PurchasedProductOptionType {
+    label: String!
+    value: String!
+}
+
+type PurchasedProductType implements Node {
+    id: ID! # Proposed value for a UUID that can help us search for the product in the backend without depending on productId, variantId and options to add this product to cart.
+    name: String!
+    sku: String!
+    price: Money!
+    imageUrl: String!
+    lastOrderedAt: DateTime!
+    options: [PurchasedProductOptionType!]!
+}
+
+type PurchasedProductEdge {
+    node: PurchasedProductType!
+    cursor: String!
+}
+
+type PurchasedProductConnection {
+    edges: [PurchasedProductEdge!]!
+    pageInfo: PageInfo!
+    collectionInfo: CollectionInfo
+}
+
+input PurchasedProductDateRangeFilterInput {
+    from: DateTime
+    to: DateTime
+}
+
+input PurchasedProductsSearchInput {
+    orderId: ID
+    search: String
+    dateRange: PurchasedProductDateRangeFilterInput
+    before: String
+    after: String
+    first: Int
+    last: Int
+}
+
+input ConfiguredProductInput {
+    configuredProductId: ID! # shopping-list/adasd-2132e-dwad2 or purchased/as2d2-dd3gv
+    quantity: Int!
+}
+
+input ProductValidationInput {
+    sku: String!
+}
+
+input BulkProductValidationInput {
+    products: [ProductValidationInput!]! 
+}
+
+type ProductValidationErrorType {
+    lineNumber: Int!
+    sku: String!
+    field: String!
+    message: String! # Can we get a translated string from the backend? Or should we send an error type for the frontend
+}
+
+type BulkProductValidationResult {
+    success: Boolean! # Do we need this ? or assume empty errors means success
+    errors: [ProductValidationErrorType!]!
+}
+
+input AddConfiguredProductsDataInput {
+    configuredProducts: [ConfiguredProductInput!]!
+}
+
+input AddConfiguredProductsToCartInput {
+    cartId: ID! # should we make this optional if its for a new cart?
+    data: AddConfiguredProductsDataInput!
+}
+
+type AddConfiguredProductsToCartError implements Error {
+    message: String!
+}
+
+type AddConfiguredProductsToCartResult {
+    cart: Cart # AddCartLineItemsResult also returns cart
+    errors: [AddConfiguredProductsToCartError!]! # Error only contains message: string.
+}
+
+input AddQuoteLineItemsDataInput {
+    lineItems: [CartLineItemInput!]!
+}
+
+input AddQuoteLineItemsInput {
+    # ignoring quote id, since draft quotes are unique per user
```

Comment:
as we discussed in slack / standup - let's support multiple draft quotes, so `quoteId` should be required.